util/openssl-update-copyright

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
As you use bashisms, this should be `#!/bin/bash`

Reply:
```#!/bin/bash``` is not portable; how about ```/usr/bin/env bash```?